src/ImageSharp.Web/Format.cs

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Six Labors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Web
+{
+    /// <summary>
+    /// Contains reusable static instances of known image formats.
+    /// </summary>
+    public static class Format
+    {
+        /// <summary>
```

Comment:
What about others like APNG, AVIF and even SVG?

Reply:
You can create new `FormatCommand` instances for additional raster types to pass as a parameter. This static class simply holds known values. 

Note: We'd never support SVG since it is not a raster image format.